www/source/tutorials/build-your-own/next-steps.html.slim

```diff
@@ -0,0 +1,17 @@
+---
+title: Habitat - Next Steps
+---
+
+section
+ h1 Next Steps
+
+ p Now that you have completed the web application guide, you are ready to learn more about what Habitat can provide to you. We encourage you to read through the following topics for more information:
```

Comment:
Next steps: 

- Check out these articles 
- Check out these blogs
- Join our slack channel to ask questions and chat with the core team
- Subscribe to our forums
- Read our roadmap 
- Check out our github repo 
- Check out our github project tracker